tools/src/main/java/org/apache/kafka/tools/streams/StreamsGroupCommandOptions.java

```diff
@@ -23,18 +23,25 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 
 import joptsimple.OptionSpec;
 
+import static org.apache.kafka.tools.ToolsUtils.minus;
+
 public class StreamsGroupCommandOptions extends CommandDefaultOptions {
     public static final Logger LOGGER = LoggerFactory.getLogger(StreamsGroupCommandOptions.class);
 
     public static final String BOOTSTRAP_SERVER_DOC = "REQUIRED: The server(s) to connect to.";
     public static final String GROUP_DOC = "The streams group we wish to act on.";
     public static final String LIST_DOC = "List all streams groups.";
     public static final String DESCRIBE_DOC = "Describe streams group and list offset lag related to given group.";
+    private static final String ALL_GROUPS_DOC = "Apply to all Streams groups.";
```

Comment:
nit: It seems to me that the capitalization which we are using in general is "streams groups" rather than "Streams groups". I know that this is being used to support Kafka Streams, but I still contend that "streams groups" is better. Please could you align the capitalization across all messages in this tool. Thank you.